requirements.txt

```diff
@@ -1,9 +1,10 @@
 click==7.1.2
-Jinja2==2.11.2
+Jinja2==2.11.3
 PyYAML==5.3.1
 awscli==1.18.182
 requests==2.25.0
 boto3==1.16.56
 docopt==0.6.2
 bcrypt==3.2.0
 yq==2.11.1
+rsa>=4.7 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Nice. But is this going to get lost next time we update dependencies manually?

Reply:
No I think it will be fine